source/views/building-hours/report/editor.tsx

```diff
@@ -106,7 +105,7 @@ function WeekTogglesIOS(props: WeekTogglesProps) {
 
 	let allDays: DayOfWeekEnumType[] = ['Mo', 'Tu', 'We', 'Th', 'Fr', 'Sa', 'Su']
 
-	return (
+	let WeekTogglesIOS = (
```

Comment:
```suggestion
   if (Platform.OS === 'ios') {
      return (…)
   } else if (Platform.OS === 'android') {
      return (…)
   } else {
      return null 
   } 
```

Thoughts?

Reply:
Nice, let's go with that.

Reply:
Is this a good case for Platform.select?

```js
return Platform.select({
	ios: (
		<Row style={styles.weekToggles}>
		{allDays.map((day) => (
			<ToggleButton
				key={day}
				active={props.days.includes(day)}
				onPress={toggleDay}
				text={day}
			/>
		))}
		</Row>
	),
	android: (
		<View>
			{allDays.map((day, i) => (
				<View key={day}>
					<CellToggle
						key={day}
						label={day}
						onChange={() => toggleDay(day)}
						value={days.includes(day)}
					/>
					{i === allDays.length - 1 ? null : <ListSeparator force={true} />}
				</View>
			))}
		</View>
	)
})
```

Reply:
Sure!